src/libp2pConfigs/getDhtLibp2pConfig.ts

```diff
@@ -8,43 +8,10 @@ export const getDhtLibp2pConfig = (): Libp2pOptions => ({
   dht: kadDHT({
     clientMode: true
   }),
-  /**
-     * @see https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#configuring-connection-manager
-     */
-  connectionManager: {
-    maxConnections: Infinity,
-    minConnections: 1,
-    pollInterval: 2000
-    // autoDial: true
-    // addressSorter: (addressA, addressB) => {
-    //   // Sort addresses by valid browser protocols first
-    //   const addressAString = addressA.multiaddr.toString()
-    //   const addressBString = addressB.multiaddr.toString()
-    //   const addressAIsValidBrowserProtocol = validTransports.some((transport) => addressAString.includes(transport))
-    //   const addressBIsValidBrowserProtocol = validTransports.some((transport) => addressBString.includes(transport))
-    //   if (addressAIsValidBrowserProtocol && !addressBIsValidBrowserProtocol) {
-    //     return -1
-    //   }
-    //   if (!addressAIsValidBrowserProtocol && addressBIsValidBrowserProtocol) {
-    //     return 1
-    //   }
-    //   return 0
-    // }
-  },
-  /**
-     * @see https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#configuring-peerstore
-     */
-  peerRouting: { // Peer routing configuration
-    refreshManager: { // Refresh known and connected closest peers
-      enabled: true, // Should find the closest peers.
-      interval: 6e5, // Interval for getting the new for closest peers of 10min
-      bootDelay: 10e3 // Delay for the initial query for closest peers
-    }
```

Comment:
These are all set to the default values, overriding like this does nothing.

Reply:
+1. Mostly just learning process for me where I copied these over and was doing some experimentation